### ISSUE TITLE
Gesture typing doesn't work

### ISSUE THREAD

**Issue by Swpolo** [kiideveloper/kiikeyboard2#2]
Hello,
The gesture typing doesn't work on my phone, even though it works with KiiKeyboard 1.
Current gesture typing options states are :
- Enable Gesture Typing : ON
- Dynamic floating preview : OFF
- Show all predictions : OFF
- Show gesture trail : ON
- Gesture trail thickness : Portait - 100% / Landscape : 100%
- Phrase gesture : ON

It tried it on two differents devices but is hasn't worked on any.
ASUS Zenfone 2 Z551ML with Android 5.0 (Rooted).
Samsung Galaxy S4 Mini GT-I9195 with Android 4.4.2 (Rooted).

**Comment by kiideveloper** [comment]
This is a known issue, gesture typing is not yet working on ASUS zenfone. Are you sure Kii Keyboard 1's gesture typing works on zenfone?

**Comment by Swpolo** [comment]
Yes, Kii Keyboard 1's gesture typing definitely works on my Zenfone2.

**Comment by kiideveloper** [comment]
It's also odd that it doesn't work on Galaxy s4 Mini, gesture typing should work there. This is odd, can you download https://play.google.com/store/apps/details?id=com.kritikalerror.cpuinfo and paste the info here?

**Comment by Swpolo** [comment]
CPU Info:
CPU ABI: armeabi-v7a
Processor : ARMv7 Processor rev 0 (v7I)
processor : 0
BogoMIPS : 13.53

processor : 1
BogoMIPS : 13.53

Features : swp half thumb fastmult vfp edsp neon
vfpv3 tls vfpv4
CPU implementer    : 0x51
CPU architecture: 7
CPU variant : 0x2
CPU part    : 0x04d
CPU revision : 0

Hardware : SAMSUNG SERRANO
Revision   : 0005
Serial    : 00009090000d1c5


Screenshot : http://imgur.com/LjESDi2

**Comment by Swpolo** [comment]
I switch the option off and on again on the samsung S4 mini, and it now works.
